tmk_core/protocol/ps2_mouse.c

```diff
@@ -157,6 +157,21 @@ static inline void ps2_mouse_convert_report_to_hid(report_mouse_t *mouse_report)
     // invert coordinate of y to conform to USB HID mouse
     mouse_report->y = -mouse_report->y;
 #endif
+
+#ifdef PS2_MOUSE_ROTATE
+    int8_t x = mouse_report->x;
+    int8_t y = mouse_report->y;
+#if PS2_MOUSE_ROTATE == 90
+    mouse_report->x = y;
+    mouse_report->y = -x;
+#elif PS2_MOUSE_ROTATE == 180
+    mouse_report->x = -x;
+    mouse_report->y = -y;
+#elif PS2_MOUSE_ROTATE == 270
+    mouse_report->x = -y;
+    mouse_report->y = x;
+#endif
```

Comment:
Thanks for your feedback!  However, I'm following the style of the rest of the file and the published coding conventions. From https://docs.qmk.fm/#/coding_conventions_c:

> You should match the style of the code surrounding your change
> Do not put whitespace between # and if

I can make the changes you've suggested if that's recommended, but in that case the docs should be updated, and perhaps it would be better to reformat the whole project in another PR.

Reply:
There is a discrepancy between the coding conventions and the current `clang-format` configuration, for reasons discussed in #8664. I will look into updating the doc to address this.

For the time being, the indentation should be made to match the rest of the file, as you said. However, since this file doesn't actually have any preexisting nested preprocessor directives, there's nothing to go off of. If we look at other files in the same directory (e.g. [`ps2_mouse.h`](https://github.com/qmk/qmk_firmware/blob/56e36e4935d0e14e75bf78052afe916d5573df9f/tmk_core/protocol/ps2_mouse.h)), however, we see that they use the “space before hash” style since they were formatted by `clang-format`. Which is why I suggested that it be used here, as well.

Reply:
Thanks for the explanation!